### PR TITLE
Add Dispute resource

### PR DESCRIPTION
**Pull request by remear**

### CODE REVIEW

**Comment by remear** [comment]
ping @matthewfl

**Comment by matthewfl** [comment]
can we set a limit for how long it polls for disputes?

**Comment by remear** [comment]
Sure. If we add a limit, the events test should probably have one as well.

**Comment by matthewfl** [comment]
yes they should both have one, I just don't want these tests to run for >1 hour on travis while no one is watching

**Comment by remear** [comment]
Agree, however, I also would like to see test disputes take a more reasonable and reliable amount of time to populate.

File: test/test.js [hunk 1]
```diff
@@ -97,15 +101,46 @@ test('add_bank_account_to_customer', function(assert, bank_account_create, custo
...
+test('debit_card', function (add_card_to_customer) {
     return add_card_to_customer.debit({amount: 5000});
 });
 
-
 test('hold_card', function (add_card_to_customer) {
     return add_card_to_customer.hold({amount: 400});
 });
 
+test('dispute', function (cb, assert, marketplace) {
```
Comment (matthewfl) [inline test/test.js:112]: 1

File: test/test.js [hunk 2]
```diff
@@ -309,11 +344,55 @@ test('credit_create_bank_account', function (cb, assert, marketplace, debit_card
     });
 });
 
+test('dispute', function (cb, assert, marketplace) {
```
Comment (matthewfl) [inline test/test.js]: 2

**Pull request merged**